survey/survey-web-adaptor/src/main/java/me/nalab/survey/web/adaptor/updatetarget/TargetPositionUpdateController.java

```diff
@@ -0,0 +1,30 @@
+package me.nalab.survey.web.adaptor.updatetarget;
+
+import javax.validation.Valid;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.RequestAttribute;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.RequiredArgsConstructor;
+import me.nalab.survey.application.port.in.web.target.update.TargetPositionUpdateUseCase;
+import me.nalab.survey.web.adaptor.updatetarget.request.TargetPositionUpdateRequest;
+
+@RestController
+@RequestMapping("/v1")
+@RequiredArgsConstructor
+public class TargetPositionUpdateController {
+
+	private final TargetPositionUpdateUseCase targetPositionUpdateUseCase;
+
+	@PatchMapping("/users")
+	public ResponseEntity<Void> updateTargetPosition(@RequestAttribute("logined") Long loginId,
+		@Valid @RequestBody TargetPositionUpdateRequest targetPositionUpdateRequest) {
```

Comment:
```suggestion
		@RequestBody TargetPositionUpdateRequest targetPositionUpdateRequest) {
```

이거 권한 검증이 필요없을거 같아요.
token 복호화해서 그거 id에 해당하는 target에 포지션을 적용하는거라, 자신이 권한을 갖고있는지 검증하는 로직을 탈 필요가 없을거 같다?